packages/SidePanel/src/components/Overlay/Overlay.js

```diff
@@ -10,15 +10,13 @@ const propTypes = {
    * Will accept the click event on outside of the sidepanel when losing focus
    * @boolean
    */
-  background: PropTypes.string,
```

Comment:
I think this technically constitutes a breaking change to `SidePanel`.  I'm okay with that, it will be very low impact, but please indicate that in the PR description, tag the PR with that label, and add a note to the `CHANGELOG`.